ci-operator/config/terraform-redhat/terraform-provider-rhcs/OWNERS

```diff
@@ -25,3 +26,5 @@ reviewers:
 - robpblake
 - sagidayan
 - vkareh
+- shellymiron
```

Comment:
I think its generated from the the OWNERS file in the repository
I will approve it, lets see if it going to be override